core/api/src/main/java/com/alipay/sofa/rpc/common/RpcOptions.java

```diff
@@ -357,6 +357,18 @@
      * @since 5.2.0
      */
     public static final String CONSUMER_REPEATED_REFERENCE_LIMIT        = "consumer.repeated.reference.limit";
+    /**
+     * 初始化连接时建立连接的百分比
+     *
+     * @since 5.4.1
+     */
+    public static final String CONSUMER_ELASTICCONNECT_PRECENT          = "consumer.connect.elastic.precent";
```

Comment:
key and value may keep same,like this

public static final String CONSUMER_CONNECT_ELASTIC_PRECENT          = "consumer.connect.elastic.precent";